libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -99,7 +99,7 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
   }
 
   @Input() set minDate(value: Date) {
-    if (value && this.activeMonth.toDate() < value) {
+    if (value && this.activeMonth && this.activeMonth.isBefore(value)) {
```

Comment:
Could the set a default value for activeMonth eg. current month?

Reply:
@lydemann We might look into this later, but for now that's not the quickest fix ;)